From ad6216416e5142376c67acc425616adc394f1270 Mon Sep 17 00:00:00 2001
From: Bharat Kumar Gogada <bharat.kumar.gogada@xilinx.com>
Date: Mon, 13 Jan 2020 15:26:26 +0530
Subject: [PATCH 0830/1852] PCI: XDMA PL PCIe: Remove unused platform data

commit 9f0d55e05299ecbd7ccefc360fb46e39b5afeac1 from
https://github.com/Xilinx/linux-xlnx.git

Driver is not using platform drvdata pointer, so
no need to set it.

Signed-off-by: Bharat Kumar Gogada <bharat.kumar.gogada@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: not-upstreamable
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/pci/controller/pcie-xdma-pl.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/pci/controller/pcie-xdma-pl.c b/drivers/pci/controller/pcie-xdma-pl.c
index 6e6837a99257..05511376018b 100644
--- a/drivers/pci/controller/pcie-xdma-pl.c
+++ b/drivers/pci/controller/pcie-xdma-pl.c
@@ -654,7 +654,6 @@ static int xilinx_pcie_probe(struct platform_device *pdev)
 	list_for_each_entry(child, &bus->children, node)
 		pcie_bus_configure_settings(child);
 	pci_bus_add_devices(bus);
-	platform_set_drvdata(pdev, port);
 	return 0;
 
 error:
-- 
2.31.1

